99-Swiss officer ratios.ipynb

```diff
@@ -0,0 +1,959 @@
+{
```

Comment:
<p>Title is a bit confusing.</p><p>Number of unique ratios, first five ratios -&gt; textbox in plot (please round numbers)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gabrieltorresgamez/fhnw-san-project/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>